src/toil/test/wdl/wdltoil_test.py

```diff
@@ -332,12 +342,77 @@ def mock_get_transitive_dependencies(self: Any, node_id: str) -> Set[str]:
                     }
 
                     result = WDLSectionJob.coalesce_nodes(["decl1", "decl2", "successor"], WDLWorkflowGraph([]))
-                    
+
                     assert len(result) == 2
                     assert "decl1" in result[0]
                     assert "decl2" in result[0]
                     assert "successor" in result[1]
 
 
+@integrative
+@slow
+@pytest.mark.timeout(600)
+class WDLKubernetesClusterTest(AbstractClusterTest):
```

Comment:
I think this test needs to be specifically added to the list of tests in the `provisioner_integration` step in `.gitlab-ci.yml` in order for it to actually get run on CI.